gallery/src/main.scss

```diff
@@ -128,6 +128,10 @@
     white-space: normal;
   }
 
+  .nx-list__item {
+    align-items: flex-start;
```

Comment:
A better alignment can be achieved by overriding `align-items` to `first baseline` on the `.nx-radio-checkbox` instead of this.

Reply:
> A better alignment can be achieved by overriding `align-items` to `first baseline` on the `.nx-radio-checkbox` instead of this.

Very much not better IMO 
<img width="187" alt="Screen Shot 2022-04-05 at 2 15 18 PM" src="https://user-images.githubusercontent.com/1991443/161822736-f6442629-574a-4806-a122-c8c3b73a8a39.png">

Reply:
> Very much not better IMO

I think you must not have done quite what I meant.  It looks like the following for me.  Note that the checkbox text and the link are baseline-aligned which is not the case in this PR currently. 
![image](https://user-images.githubusercontent.com/3630091/161823718-9b4406ec-ceb2-48a9-829b-db12467fb341.png)

Reply:
> > Very much not better IMO
> 
> I think you must not have done quite what I meant. It looks like the following for me. Note that the checkbox text and the link are baseline-aligned which is not the case in this PR currently. ![image](https://user-images.githubusercontent.com/3630091/161823718-9b4406ec-ceb2-48a9-829b-db12467fb341.png)

To my eye this is six of one, half a dozen of the other but I fixed the problem above at the same time so: https://github.com/sonatype/sonatype-react-shared-components/pull/681/commits/2d426353c77a62d6762e9df25208442f380cfa7e